documentation/docs/tutorials/value-transactions/01-introduction.md

```diff
@@ -0,0 +1,50 @@
+---
+description: "Get acquainted with the structure and functionality of transactions in IOTA's Stardust protocol in the
```

Comment:
```suggestion
description: "Learn the structure and functionality of transactions in IOTA's Stardust protocol in the
```

A slightly shorter version.

Reply:
It should be Lean about the structure, which isn't that much shorter.